Classes/Job/StaticMethodCallJob.php

```diff
@@ -95,9 +95,13 @@ public function getLabel()
         $arguments = [];
         foreach($this->arguments as $argumentValue) {
             if (TypeHandling::isSimpleType(gettype($argumentValue))) {
-                $arguments[] = gettype($argumentValue);
+                if (is_array($argumentValue)) {
+                    $arguments[] = gettype($argumentValue);
+                } else {
+                    $arguments[] = $argumentValue;
```

Comment:
That may lead to pretty long outputs if it was a longer string for example. Even a float might be a bit long already. And bools/null values are also probably not represented nicely.

Reply:
Thats right, I'm missing some suggestions what to do else, the code just changed to work again, the logic to display the content is still the same, beside array handling. What about casting the `$argumentValue` to string and doing a substring(0,10)  and add dots (...) in the case the string representation was longer than this `